src/lib/times.js

```diff
@@ -77,7 +78,8 @@ export function confirmationTime(time, timeType) {
   if (upcoming.minutes()) {
     timeStr = getLocalizedDateTime(upcoming, 'confirmation_time');
   }
-  const dateStr = getLocalizedDateTime(upcoming, 'confirmation_date');
+  const dateStr = getLocalizedDateTime(upcoming,
+    (upcoming.year() === thisYear) ? 'confirmation_date' : 'confirmation_date_with_year');
```

Comment:
Well, I was trying to avoid re-running `getLocalizedDateTime` since all that changed was the name of the string. But, I can use an if statement instead